src/rpc.cpp

```diff
@@ -133,6 +134,27 @@ RPCServer::RPCServer() {
         RPCObjectID realm_id = store_object(realm_object);
         return (json){{"result", realm_id}};
     };
+    m_requests["/create_user"] = [this](const json dict) {
```

Comment:
Objects are stored when they are serialized before they are returned. Not sure why this wouldn't just work - although I haven't looked at this code for quite a while.